Runtime/Extensions/ULongExtensions.cs

```diff
@@ -0,0 +1,21 @@
+namespace FishingCactus
+{
+    public static class ULongExtensions
+    {
+        public static uint GetToggledFlagsCount(
+            this ulong flags
+            )
+        {
+            uint count = 0;
+
+            while( flags != 0 )
+            {
+                flags &= flags - 1;
+
+                ++count;
```

Comment:
Would be best to use bitwise operators still for the same reasons as for where it was before ^^^

Reply:
That being only using them:
```cs
count += flags & 1;
flags >>= 1.
```

Reply:
I reverted as `This code is very efficient as it only iterates once for each bit rather than once for every possible bit as in the other examples` (https://stackoverflow.com/a/1333267/12806660)